cmake/modules/platform/os/zos.cmake

```diff
@@ -32,17 +32,17 @@ list(APPEND OMR_PLATFORM_DEFINITIONS
 	-DZOS
 )
 
-list(APPEND OMR_PLATFORM_INCLUDE_DIRECTORIES
-	${CMAKE_SOURCE_DIR}/util/a2e/headers
-	/usr/lpp/cbclib/include
-	/usr/include
-)
-
 # Create helper targets for specifying ascii/ebcdic options
 add_library(omr_ascii INTERFACE)
 target_compile_definitions(omr_ascii INTERFACE -DIBM_ATOE)
-target_compile_options(omr_ascii INTERFACE "-Wc,convlit(ISO8859-1)")
+target_compile_options(omr_ascii INTERFACE "-Wc,convlit(ISO8859-1),nose")
 target_link_libraries(omr_ascii INTERFACE j9a2e)
+target_include_directories(omr_ascii INTERFACE
+	# OMR might not be included yet, so we can't reference OMR_SOURCE_DIR.
+	${CMAKE_CURRENT_LIST_DIR}/../../../../util/a2e/headers
```

Comment:
This seems fragile. Where is `OMR_SOURCE_DIR` defined? Is there any way for that to be available here?

Reply:
`omr_SOURCE_DIR` (note comment is a typo, and should be `omr_` ) gets defined automatically by `project(omr ....)`.  However for consumers of OMR which want to reuse our platform detection code, they will be including this code, before their `add_subdirectory()` of omr. As for manually introducing a definition, the only possibility I can think of essentially boils down to the same thing. eg
```
set(omr_SOURCE_DIR "${CMAKE_CURRENT_LIST_DIR}/../../../..") 
...
```

Reply:
Ok, thanks for the explanation. I'm fine with the change.